Update README.md with link to wiki minutes page

wrathwolf opened the pull request:
#### What type of PR is this?

* documentation

wrathwolf committed:
Update README.md with link to wiki minutes page

hdamker commented on README.md:
```diff
@@ -11,7 +11,6 @@
 
 Sandbox API Repository to describe, develop, document, and test the IoTNetworkOptimization Service API(s). The repository does not yet belong to a CAMARA Sub Project.
 
-* API Repository [wiki page](https://lf-camaraproject.atlassian.net/) !! not yet created, update with final link
```
Why deleted? The line is part of the template and should stay -- with updated link, here: https://lf-camaraproject.atlassian.net/wiki/x/AYCzBw

hdamker reviewed (changes requested):
@wrathwolf One point for (keeping) the alignment with the template

FabrizioMoggio committed:
Link to Wiki

according to: https://github.com/camaraproject/IoTNetworkOptimization/pull/8#discussion_r2119531382
